### DISCUSSION TITLE
Add Feature

### DISCUSSION THREAD

**Discussion by nissanskyliner34g** [namidaco/namida#330] [Ideas]
Hello
Please add auto fill for music that doesn't have music data like title or artist album year of music

**Comment by MSOB7YY** [comment]
by default, the artist & title are extracted from filename when there are no tags.

if you mean manually fetch from online services, then alr requested at #5

**Comment by worthyknight** [comment]
could you please add a landscape mode

**Reply by MSOB7YY** [reply]
not any soon #202
